.github/workflows/mgrpush-unit-tests.yml

```diff
@@ -0,0 +1,27 @@
+name: Run unit tests for mgr-push
+
+on:
+  pull_request:
+    paths: 'client/tools/mgr-push/*'
+
+jobs:
+  mgrpush_tests:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out repository code
+        uses: actions/checkout@v3
+
+      - name: Setup Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: "3.x"
+
+      - name: Install Python dependencies
+        run: |
+          python3 -m pip install --upgrade pip
+          pip install pytest mock pyyaml pyOpenSSL python-debian
```

Comment:
In the future we should think about how we want to manage Python dependencies. That's not in the scope of this PR, but we need to define something that we then follow everywhere. Right now we maintain them in too many places.

Reply:
Do you think it should be done differently at the moment?

Reply:
No, we don't need to change it now.